code/go/0chain.net/smartcontract/minersc/sc.go

```diff
@@ -106,9 +106,11 @@ func (msc *MinerSmartContract) setSC(sc *sci.SmartContract, bcContext sci.BCCont
 	msc.SmartContract.RestHandlers["/getNodepool"] = msc.GetNodepoolHandler
 	msc.SmartContract.RestHandlers["/getUserPools"] = msc.GetUserPoolsHandler
 	msc.SmartContract.RestHandlers["/getMinerList"] = msc.GetMinerListHandler
+	msc.SmartContract.RestHandlers["/get_miner_lat_long"] = msc.GetMinerGeolocationsHandler
 	msc.SmartContract.RestHandlers["/get_miners_stats"] = msc.GetMinersStatsHandler
 	msc.SmartContract.RestHandlers["/get_miners_stake"] = msc.GetMinersStateHandler
 	msc.SmartContract.RestHandlers["/getSharderList"] = msc.GetSharderListHandler
+	msc.SmartContract.RestHandlers["/get_sharder_lat_long"] = msc.GetSharderGeolocationsHandler
```

Comment:
Use `/get_sharder_geolocation` and `/get_miner_geolocation` perhaps.

Reply:
I also prefer `/get_sharder_geolocation`, but there's a similar handler for blubber with endpoint `/get_blobber_lat_long`. so I named these handler endpoints like the blubber endpoint to maintain consistency.
If that's not an issue, I'll happy to rename them.

Reply:
done